src/sources/soundsourceffmpeg.cpp

```diff
@@ -495,317 +487,47 @@ long SoundSourceFFmpeg::seek(long filepos) {
             }
         }
 
-        if (filepos == 0) {
-            readFramesToCache((SOUNDSOURCEFFMPEG_CACHESIZE - 50), -1);
+        if (frameIndex == 0) {
+            readFramesToCache((AUDIOSOURCEFFMPEG_CACHESIZE - 50), -1);
```

Comment:
This just not to fill cache full so there is 50 places still available. Just trying not to full fill memory.